src/shard.h

```diff
@@ -9,10 +9,13 @@
 #ifndef ZMAP_SHARD_H
 #define ZMAP_SHARD_H
 
+#include <signal.h>
```

Comment:
huhwut?

Reply:
I thought I was going to need `sig_atomic_t`, but ended up not. Will remove include.